doc/Developing/Using-Git.md

```diff
@@ -39,7 +39,7 @@ git clone git@github.com:/yourusername/librenms.git
 ```
 
 If you are starting with an existing LibreNMS install either from our Virtual machine or install docs then you 
-will need to update the remote url. To do that run:
+will need to update the remote url. Be aware that daily.sh will not work properly after changing your origin. To do that run:
```

Comment:
Are you sure? I tested mine doing the set-upstream and can switch between urls and run daily.sh with no issue.

If you're correct then I'll close this as we shouldn't break daily.sh and expect people to fix it themselves.